package.json

```diff
@@ -20,12 +20,13 @@
   "license": "ISC",
   "dependencies": {
     "@ethersproject/providers": "^5.0.10",
-    "ethers": "^5.0.15"
+    "@ungap/structured-clone": "^1.2.0",
+    "ethers": "^6.6.0"
   },
   "devDependencies": {
     "@types/node": "^14.11.2",
     "ethereum-abi-types-generator": "^1.0.6",
-    "typescript": "^4.1.3"
+    "typescript": "^5.3.0"
```

Comment:
I upgraded typescript since I was running into build errors after installing 6.6, and it was set to 4.1 which is quite old.